### PR TITLE
fixes #53 improve error handling

### LINKED ISSUE #53: missing error handling during execution of tesseract
Hi guys,

if e.g. the inout is not a valid file the error messages of tesseract were taken as text, which is unwanted behavior I guess. It would be better to consider the returncode of the tesseract command and in case of errors to throw an exception.

### PR DESCRIPTION
**Pull request by luemic**

### CODE REVIEW

File: src/TesseractOCR.php [hunk 1]
```diff
@@ -80,7 +80,15 @@ public function __construct($image)
      */
     public function run()
     {
-        return trim(`{$this->buildCommand()}`);
+        $output = array();
```
Comment (thiagoalessio) [inline src/TesseractOCR.php]: not needed.

File: src/TesseractOCR.php [hunk 2]
```diff
@@ -80,7 +80,15 @@ public function __construct($image)
...
+        $returnValue = 0;
```
Comment (thiagoalessio) [inline src/TesseractOCR.php]: not needed.

**Comment by thiagoalessio** [comment]
Hi, I'm sorry but there is no way to accept that.

Look, I really appreciate your will to contribute, that is an awesome attitude and I encourage you to keep doing it, but please be more careful in the future.

First you committed an empty file, and now you just copied and pasted the test case from your previous (already declined) pull request without even running the tests before committing.

**Comment by luemic** [comment]
Well I guess I'll fork it and maintain it by myself.

**Pull request closed without merging**